components/compliance-service/migrations/migrations.go

```diff
@@ -0,0 +1,112 @@
+package migrations
+
+import (
+	"context"
+	"encoding/json"
+	"time"
+
+	"github.com/olivere/elastic/v7"
+	"github.com/sirupsen/logrus"
+)
+
+type Node struct {
+	NodeUUID  string `json:"node_uuid"`
+	EndTime   string `json:"end_time"`
+	DayLatest bool   `json:"day_latest"`
+}
+
+var setNodes map[string]Node
+
+func GetNodesDayLatestTrue(client *elastic.Client, ctx context.Context) ([]Node, error) {
+	var nodes []Node
+	termQueryDayLatest := elastic.NewTermQuery("day_latest", true)
+
+	boolQuery := elastic.NewBoolQuery().
+		Must(termQueryDayLatest)
+
+	fsc := elastic.NewFetchSourceContext(true).Include(
+		"node_uuid",
+		"end_time",
+		"day_latest",
+	)
+
+	searchSource := elastic.NewSearchSource().
+		FetchSourceContext(fsc).
+		Query(boolQuery).
+		Size(10000)
```

Comment:
@dave-augustus  This size should be greater than this, either we take up  index one by one for each day. Or this should be around 100K.